bazel/external/lightstep.BUILD

```diff
@@ -0,0 +1,38 @@
+load("@//bazel/external:helpers.bzl", "cc_genrule_deps")
+
+cc_library(
+    name = "lightstep",
+    srcs = [":_prefix/lib/liblightstep_core_cxx11.a"],
+    hdrs = glob([
+        "src/c++11/lightstep/**/*.h",
+        "src/c++11/mapbox_variant/**/*.hpp",
+    ]) + [
+        ":_prefix/include/collector.pb.h",
+        ":_prefix/include/lightstep_carrier.pb.h",
+    ],
+    includes = ["src/c++11", "_prefix/include"],
+    deps = ["@protobuf_bzl//:protobuf"],
+    visibility = ["//visibility:public"],
+)
+
+cc_genrule_deps(
+    name = "protobuf_deps",
+    deps = ["@protobuf_bzl//:protobuf"],
+)
+
+genrule(
+    name = "install",
+    srcs = glob(["**"]) + [
+        ":protobuf_deps",
+        "@protobuf_bzl//:well_known_protos",
+    ],
+    cmd = GENRULE_REPOSITORY_COMMAND,
```

Comment:
This is quite neat, but also non-obvious to the reader. Can you add a comment on each of these sites where you reference `GENRULE_REPOSITORY_COMMAND` and explain where it is coming from?

Reply:
Changed to use a function that takes a label. This should be a bit more self-describing.